CHANGES.rst

```diff
@@ -1,11 +1,12 @@
 Changes
 =======
 
-Version 0.9.1 (UNRELEASED)
+Version 0.9.1 (2023-09-27)
 --------------------------
 
-- Fixes the total job count reported to workflow controller by excluding jobs restored from the workspace and not executed by Snakemake.
+- Fixes the reported total number of jobs by excluding cached jobs that are restored from the workspace and not executed by Snakemake.
 - Fixes an issue where some workflows would be stuck waiting for already-finished jobs.
```

Comment:
- Fixes an issue where workflows could get stuck waiting for already-finished jobs.

Reply:
Also, one item missing:

- Changes remote storage fie support to use XRootD 5.6.0.